src/main/java/software/amazon/smithy/plugin/language/SmithySyntaxHighlighter.java

```diff
@@ -0,0 +1,279 @@
+            createTextAttributesKey("SMITHY_NODE_OBJECT_TYPE_NAME", DefaultLanguageHighlighterColors.KEYWORD);
+    public static final TextAttributesKey NODE_STRING_VALUE =
+            createTextAttributesKey("SMITHY_NODE_STRING_VALUE", DefaultLanguageHighlighterColors.PARAMETER);
+    public static final TextAttributesKey NODE_VALUE =
+            createTextAttributesKey("SMITHY_NODE_VALUE", DefaultLanguageHighlighterColors.PARAMETER);
+    public static final TextAttributesKey NUMBER =
+            createTextAttributesKey("SMITHY_NUMBER", DefaultLanguageHighlighterColors.NUMBER);
+    public static final TextAttributesKey PLUS =
+            createTextAttributesKey("SMITHY_PLUS", DefaultLanguageHighlighterColors.COMMA);
+    public static final TextAttributesKey QUOTED_TEXT =
+            createTextAttributesKey("SMITHY_QUOTED_TEXT", DefaultLanguageHighlighterColors.PARAMETER);
+    public static final TextAttributesKey RBRACE =
+            createTextAttributesKey("SMITHY_RBRACE", DefaultLanguageHighlighterColors.BRACES);
+    public static final TextAttributesKey RBRACK =
+            createTextAttributesKey("SMITHY_RBRACK", DefaultLanguageHighlighterColors.BRACKETS);
+    public static final TextAttributesKey RPAREN =
+            createTextAttributesKey("SMITHY_RPAREN", DefaultLanguageHighlighterColors.PARENTHESES);
+    public static final TextAttributesKey SHAPE_ID_MEMBER =
+            createTextAttributesKey("SMITHY_SHAPE_ID_MEMBER", DefaultLanguageHighlighterColors.IDENTIFIER);
+    public static final TextAttributesKey SIMPLE_TYPE_NAME =
+            createTextAttributesKey("SMITHY_SIMPLE_TYPE_NAME", DefaultLanguageHighlighterColors.KEYWORD);
+    public static final TextAttributesKey SPACE =
+            createTextAttributesKey("SMITHY_SPACE", DefaultLanguageHighlighterColors.COMMA);
+    public static final TextAttributesKey TEXT_BLOCK =
+            createTextAttributesKey("SMITHY_TEXT_BLOCK", DefaultLanguageHighlighterColors.PARAMETER);
+    public static final TextAttributesKey TRAIT_ID =
+            createTextAttributesKey("SMITHY_TRAIT_ID", DefaultLanguageHighlighterColors.COMMA);
+    public static final TextAttributesKey USE_KEYWORD =
+            createTextAttributesKey("SMITHY_USE_KEYWORD", DefaultLanguageHighlighterColors.METADATA);
+    public static final TextAttributesKey VERSION =
+            createTextAttributesKey("SMITHY_VERSION", DefaultLanguageHighlighterColors.METADATA);
+    public static final TextAttributesKey VERSION_SECTION =
+            createTextAttributesKey("SMITHY_VERSION_SECTION", DefaultLanguageHighlighterColors.METADATA);
+    public static final TextAttributesKey WS =
+            createTextAttributesKey("SMITHY_WS", DefaultLanguageHighlighterColors.PARAMETER);
+
+    private static final TextAttributesKey[] APPLY_KEYS = new TextAttributesKey[]{APPLY};
+    private static final TextAttributesKey[] BAD_CHAR_KEYS = new TextAttributesKey[]{BAD_CHARACTER};
```

Comment:
Nit: `BAD_CHAR_KEYS` is the only shortened name for these, can just keep it similar.